packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -65,15 +65,12 @@ function SelectionTool(props: Props) {
         return;
       }
 
-      const { unprojectedPoint, sourceEvent } = evt;
-      const point = worldToData(boundPointToFOV(unprojectedPoint, camera));
-      setEndPoint(point);
+      const { worldPt, sourceEvent } = evt;
+      const endPoint = worldToData(boundPointToFOV(worldPt, camera));
```

Comment:
I think it was called `point` to avoid name-collision with the state value `endPoint`

Reply:
Oh well spotted, I'll revert the renaming.

Reply:
I've pushed a commit to enable the `no-shadow` ESLint rule. It's disabled in eslint-config-galex because it is considered opinionated. I think it's worth enabling, as I've explained in a comment. It led me to fixing a number of offences.